src/main/java/org/opennms/test/system/api/TestEnvironmentBuilder.java

```diff
@@ -135,7 +135,9 @@ public TestEnvironmentBuilder useExisting(boolean useExisting) {
 
     public TestEnvironmentBuilder all() {
         opennms();
-        minion();
+        minion1();
```

Comment:
Can we avoid always creating 3 Minions? Tests that current use the 'all()' environment don't need them.